src/nidmm/metadata/enums_addon.py

```diff
@@ -8,4 +8,6 @@
 enums_codegen_method = {
 }
 
-
+enums_additional_enums = {
+    'DigitsResolution': {},  # Just delete this enum. Otherwise we have to make it so that it will generate valid code
```

Comment:
please add more context to this document... something like
```
# Just delete this enum, client will have to pass a float. Otherwise the constants start with a number and we have to deal with it. Not worth it.
```

Reply:
We were already setting it to not code gen so the clients would already have to pass a float.

I am setting it to {} here so it will be deleted completely. Otherwise we'd have to set the names to something that wouldn't cause issues when we remove common suffixes. `_5` is common with all, but `_5_5` becomes empty when removing `_5`, thus breaking things before we are even able to say don't codegen it.